system/validator_revard/validation.md

```diff
@@ -0,0 +1,275 @@
+
+**With Polkadot-js**
+
+You need two keys for this, one to be the `controller` and one as the `stash`. The latter holds the stake and must sign at least once to "delegate" to the `controller` which is running the "day to day" operations.
+
+Assuming you are fully synched, your node is running, and you have&#x20;
+
+**Steps:**
+
+1. Go to the "staking actions" tab - "Network" -> "Staking" -> "Accounts", and click the "+ Validator" button in the top right corner.
+2. Select a `stash` and `controller` account from the dropdown, set the "value bonded" as the amount you want to stake, and choose a "payment destination", then hit "next".
+3. Paste in the public session keys (`0xabc...123`), choose a "reward commission percentage" and whether you want to allow nominations or not, then click "Bond & Validate".
+
+If you are preparing this for later, click the "+ Stash" button instead. This allows you to wait for your session keys and/or synching your node.
+
+#### Being a Validator
+
+Assuming the transaction went through, you will now appear under the "waiting" tab [here](https://polkadot.js.org/apps/#/staking). That means you are in the queue for joining the validator set, but when (and whether) you actually join depends on the competition for getting a slot.
+
+At all times, there is a limit to how many can become validators. What that number is set by the council. The current value can be found in the [chain state](https://polkadot.js.org/apps/#/chainstate) -> "staking" -> "validatorCount".
+
+Suppose that number is `n`, and that there are `m` validators that, towards the end of each `era` were already validating or joined the queue:
+
+* If `n >= m`, all will be elected
+* If `n<m`, the `n` validators with the highest **total active stake** will be elected for the upcoming `era`
+
+_Notes:_
+
+* An `era` lasts \~6h.
+* **total active stake** refers to the active stake for the validator itself, plus all of their nominators.
+
+**Transaction Rejected**
+
+There is a minimum threshold of funds required to stake as a validator. As you can get slashed, that means you can not "re-use" tokens that are staked for other "slashable" purposes, such as role stake.
+
+That number can be found in the [chain state](https://polkadot.js.org/apps/#/chainstate) -> "staking" -> "minValidatorBond". (In base value `HAPI`, meaning `1*10^-10 JOY`)
+
+Another reason the transaction can be rejected is if the maximum number of `bonded` accounts have declared as validators. That number can be found in the [chain state](https://polkadot.js.org/apps/#/chainstate) -> "staking" -> "maxValidatorsCount". How many there are currently can be found in the [chain state](https://polkadot.js.org/apps/#/chainstate) -> "staking" -> "counterForValidators".
+
+There are of course a limitless amount of other reasons the transaction could be rejected if you constructed it yourself in the cli.
+
+### Advanced Setup
+
+`TODO`
```

Comment:
It's a good opportunity to at least link our setup guide on the community repo ([storage](https://github.com/Joystream/community-repo/tree/master/working-groups/storage-group/NodeSteup/joystream-node) and [distribution](https://github.com/Joystream/community-repo/tree/master/working-groups/distributors/NodeSteup/joystream-node)) maintained by leads.

Also this while file should be renamed README.md